src/apps/loan-list/materials/utils/checkbox.tsx

```diff
@@ -24,7 +24,7 @@ const CheckBox: FC<CheckBoxProps> = ({
       <input
         id={id}
         className="checkbox__input"
-        onChange={() => onChecked(id as FaustId)}
+        onChange={() => onChecked && onChecked(id as FaustId)}
```

Comment:
Is it not possible to declare the id type in the props as a `FaustId`?

Side note:
If you choose to move the checkbox out as a global component, you would have to make use a generic for the id type, since the id would vary between several types.
Generics and components: https://javascript.plainenglish.io/react-components-as-typescript-generic-functions-8aa83afff597